document/core/valid/instructions.rst

```diff
@@ -1753,15 +1753,17 @@ Memory Instructions
 
 * The memory :math:`C.\CMEMS[y]` must be defined in the context.
 
-* Then the instruction is valid with type :math:`[\I32~\I32~\I32] \to []`.
+* Let :math:`\X{it}~\limits` be the :ref:`memory type <syntax-memtype>` :math:`C.\CMEMS[0]`.
```

Comment:
With multi memory, the index type can't come from mems[0], but needs to be the minimum of mems[x] and mems[y].